src/collections/cursor.ts

```diff
@@ -109,20 +111,15 @@ export class FindCursor {
   async _getMore() {
     const command: {
       find: {
-        filter?: string | undefined | null,
-        projection?: string | undefined | null,
-        options?: QueryOptions | undefined | null,
-        sort?: Object | null
+        filter?: Record<string, any>,
+        options?: Record<string, any>,
+        sort?: Record<string, any>
       }
     } = {
       find: {
         filter: this.filter
       }
     };
-    // Workaround for Automattic/mongoose#13050
-    if (this.options && this.options.projection && Object.keys(this.options.projection).length > 0) {
-      command.find.projection = this.options.projection;
-    }
```

Comment:
@vkarpov15 Removal of `projection` caused a couple of projection tests to fail when I enabled them. I'm adding it  back in [this commit](https://github.com/stargate/stargate-mongoose/pull/81/commits/44052bb60b171fac901e4935e826c6859a384384).